From 7fc3eb7274351425fd240505076f2030c1788014 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 10 Jul 2019 11:27:23 -0700
Subject: [PATCH 4/6] Do not build kojivm

This is not a required service and to use it, dependencies on libvirt
are pulled in, making this a larger package than it should be.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 78322c5..831c888 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 NAME=koji
 SPECFILE = $(firstword $(wildcard *.spec))
-SUBDIRS = hub builder koji cli util www plugins vm
+SUBDIRS = hub builder koji cli util www plugins
 
 ifndef PYTHON
 export PYTHON=python2
-- 
2.22.0

